src/blockrelay/graphene.cpp

```diff
@@ -1396,7 +1396,10 @@ void SendGrapheneBlock(CBlockRef pblock, CNode *pfrom, const CInv &inv, const CM
     {
         try
         {
-            CGrapheneBlock grapheneBlock(MakeBlockRef(*pblock), mempoolinfo.nTx);
+            uint64_t nSenderMempoolPlusBlock =
+                std::max(0, (int)(GetGrapheneMempoolInfo().nTx + pblock->vtx.size() - 1)); // exclude coinbase
```

Comment:
since pblock->vtx.size() must be >=1 this must be >=0 if nTx is nonnegative.  So std::max seems unnecessary... if my reasoning is correct, perhaps a comment would be better than logic.

Reply:
Yes makes sense. Fixed